pychromecast/controllers/spotify.py

```diff
@@ -41,7 +41,8 @@ def launch_app(self):
         def callback():
             """Callback function"""
             self.send_message({"type": TYPE_STATUS,
-                               "credentials": self.access_token})
+                               "credentials": self.access_token,
+                               "expiresIn": 3600})
```

Comment:
So `expiresIn` should indicate when the token expires. This value is in seconds. It is part of the access token ad refresh token responses as per the OAuth2 spec.

```python
# Refresh token response
{
    "access_token": "ABCDEFGH",
    "expires_in": 1800,
    "refresh_token": "IJKLMNOPQRST",
    "token_type": "Bearer"
}

# Access token response
{
    "access_token": "ABCDEFGH",
    "expires_in": 1800,
    "token_type": "Bearer"
}
```

Although this PR will allow the frontend to work, it will not function correctly, and it will drop the token in 1 hour?

Reply:
I think the correct solution would be to:
 - allow expires_in to be passed in. Default expires_in to the default expires_in that Spotify returns (although this might cause weird issues in the future).
 - automatically adjust the value if we send this message later than it was passed in
 - throw an error if the token has expires